mobile/src/components/chat/components/conversations-list/components/conversation/conversation.tsx

```diff
@@ -28,9 +28,18 @@ const Conversation: FC<Props> = ({
   onPress,
 }) => {
   const messageStart = messageSenderId === currentUserId ? 'You: ' : '';
+  const messageRows = lastMessage && lastMessage.split('\n');
+  const messageEnd =
+    messageRows &&
+    (messageRows.length > 1 ||
+      messageRows[0].length > OPPONENT_MESSAGE_SHORT_LENGTH)
+      ? '...'
+      : '';
```

Comment:
```suggestion
  const messageRows = lastMessage?.split('\n');
  const messageFirstRow = messageRows?.length ? messageRows[0] : '';
  const isMessageCut = Boolean(messageRows?.length) || messageFirstRow.length > OPPONENT_MESSAGE_SHORT_LENGTH;
  const message = `${messageStart}${message.slice(0, OPPONENT_MESSAGE_SHORT_LENGTH)}${isMessageCut ? '...' : ''}`;
```

Reply:
Avoid statementents like `lastMessage && lastMessage.split('\n')`, here messageRows  can be either boolean or string and it's not good